SampleApp/DemoMapViewController.swift

```diff
@@ -34,10 +34,11 @@ class DemoMapViewController:  SampleMapViewController, MapMarkerSelectDelegate {
     setupStyleObservance()
     markerSelectDelegate = self
     let appDelegate = UIApplication.shared.delegate as! AppDelegate
-    try? loadStyleAsync(appDelegate.selectedMapStyle) { [unowned self] (style) in
-      self.styleLoaded = true
-      let _ = self.showCurrentLocation(true)
-      self.showFindMeButon(true)
+    try? loadStyleAsync(appDelegate.selectedMapStyle) { [weak self] (style) in
+      guard let unwrappedSelf = self else { print("Self is nil"); return }
```

Comment:
Hah that is debug code - had a separate issue I was trying to figure out and forgot to remove that. Will do now.